.gitignore

```diff
@@ -0,0 +1,7 @@
+/cache/clean.sh
+/content/data/
+/content/static/
+/content/noxymon/
+/themes/bootstrap/nbproject/private/
+/themes/bootstrap/
```

Comment:
could you add this theme fully? (not as submodule)
or remove it from this pull request.